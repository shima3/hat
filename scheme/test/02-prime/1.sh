#!/bin/bash
$* -e main examples/prime.sch 1000003
