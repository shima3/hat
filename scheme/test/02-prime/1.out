2 is a prime number.
3 is a prime number.
4 can be divied by 2.
5 is a prime number.
6 can be divied by 2.
7 is a prime number.
8 can be divied by 2.
9 can be divied by 3.
